packages/user-mgnt/src/features/system/handler.ts

```diff
@@ -321,6 +321,21 @@ export async function getSystemHandler(
   }
 }
 
+export async function getAllSystemsHandler() {
+  const systems: ISystemModel[] = await System.find()
+
+  return systems.map((system) => {
+    const systemName = system.name
+
+    return {
+      client_id: system.client_id,
+      name: systemName,
```

Comment:
nitpick: maybe we could just use "name: system.name" here to avoid the unnecessary repetition.